node_modules/oae-admin/manageuser/manageuser.html

```diff
@@ -46,10 +46,16 @@ <h3 id="manageuser-modal-title"><!-- --></h3>
                 </form>
 
                 <form id="manageuser-actions-form" class="tab-pane form-horizontal" role="form">
```

Comment:
I think the following modal structure would be better:

```
<form id="manageuser-actions-form" class="tab-pane form-horizontal" role="form">
  <div id="manageuser-actions-main">
    <div class="modal-body">
      ...
    </div>
    <div class="modal-footer">
      ...
    </div>
  </div>
  <div id="manageuser-actions-deleteuser">
    <div class="modal-body">
      ...
    </div>
    <div class="modal-footer">
      ...
    </div>
  </div>
</form>
```

Hiding and showing becomes a matter of switching the visible div, and they can be cleanly identified by id